qt/qt_common/map_widget.cpp

```diff
@@ -245,6 +245,14 @@ void MapWidget::Build()
 
     fragmentSrc =
         "\
+      #ifdef GL_ES\n\
+        #ifdef GL_FRAGMENT_PRECISION_HIGH\n\
+          #define MAXPREC highp\n\
+        #else\n\
+          #define MAXPREC mediump\n\
+        #endif\n\
+        precision MAXPREC float;\n\
```

Comment:
If it's used only here, then define can be avoided and this line can be used directly in if else.

Reply:
Changed.